Common/src/main/java/me/egg82/tfaplus/utils/ConfigUtil.java

```diff
@@ -0,0 +1,127 @@
+package me.egg82.tfaplus.utils;
+
+import com.authy.AuthyApiClient;
+import com.authy.api.Tokens;
+import com.google.common.collect.ImmutableSet;
+import com.rabbitmq.client.ConnectionFactory;
+import me.egg82.tfaplus.TFAAPI;
+import me.egg82.tfaplus.core.FreezeConfigContainer;
+import me.egg82.tfaplus.enums.SQLType;
+import me.egg82.tfaplus.extended.CachedConfigValues;
+import me.egg82.tfaplus.extended.Configuration;
+import ninja.egg82.service.ServiceLocator;
+import ninja.egg82.service.ServiceNotFoundException;
+import ninja.egg82.sql.SQL;
+import ninja.leaping.configurate.ConfigurationNode;
+import redis.clients.jedis.JedisPool;
+
+import javax.annotation.Nullable;
```

Comment:
Did your IDE add this? I generally don't use these annotations.

Reply:
I added it.